content/en/docs/migration/flux-v1-migration.md

```diff
@@ -181,7 +180,7 @@ Configure the reconciliation of the `deploy` dir on your cluster:
 
 ```sh
 $ flux create kustomization app \
-  --source=GitRepository/app \
+  --source=app \
```

Comment:
This does not seem to be right.

Reply:
https://github.com/fluxcd/flux2/blame/main/docs/guides/flux-v1-migration.md#L170

Reply:
It should be `GitRepository/app`

Reply:
Thanks for clarifying!